dune-project

```diff
@@ -0,0 +1,23 @@
+(lang dune 1.11)
+(name hamt)
+(generate_opam_files true)
+(maintainers "thi.suzanne@gmail.com" "rudi.grinberg@gmail.com" "marek@xivilization.net")
+(authors
+  "Thibault Suzanne"
+  "Rudi Grinberg")
+(source (github thizanne/ocaml-hamt))
+(bug_reports "https://github.com/thizanne/ocaml-hamt/issues")
+(documentation "https://thizanne.github.io/ocaml-hamt/")
```

Comment:
The doc doesn't exist yet at this address. Should we keep the entry?

Reply:
This will be necessary if we chose to cut a release using `dune-release` which requires this line and will generate documentation on Github Pages.